.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!-- Thank you for contributing a pull request! Here are a few tips to help you:
+
+1. If this is your first contribution, make sure you've read the Contribution Checklist https://developers.mattermost.com/contribute/getting-started/contribution-checklist/
+2. Read our blog post about "Submitting Great PRs" https://developers.mattermost.com/blog/2019-01-24-submitting-great-prs
+3. Take a look at other repository specific documentation at https://developers.mattermost.com/contribute
+-->
+
+#### Summary
+<!--
+A description of what this pull request does.
+-->
+
+#### Ticket Link
+<!--
+If this pull request addresses a Help Wanted ticket, please link the relevant GitHub issue, e.g.
+
+  Fixes https://github.com/mattermost/mattermost-server/issues/XXXXX
```

Comment:
nit: Does `Fixes: ` work and retain the desired functionality? Can one list multiple issues?

Separately, we had this kind of functionality enabled in JIRA for a while, but we had issues where a PR would close the ticket even though there were outstanding webapp and redux changes. Will we have that same problem here?

Reply:
> nit: Does `Fixes: ` work and retain the desired functionality?

According to https://help.github.com/articles/closing-issues-using-keywords/ `Fixes:` works fine. Why do you wan the `:`?

>  Can one list multiple issues?

You need multiple `fixed` to close multiple issues, e.g. `Fixes #X, fixes #Y`.


> Separately, we had this kind of functionality enabled in JIRA for a while, but we had issues where a PR would close the ticket even though there were outstanding webapp and redux changes. Will we have that same problem here?

That's a good point. But there is a minority of contributions which target multiple repos.

But you'r right, this is a drawback. For me it's still worth to use `fixes`.